R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
Hadley is of course right that `testthat` does not import `crayon`; more precisely, it imports it at build time. But I am actually not sure that the current solution is optimal, considering how R packages work. 

What is a bit unintuitive is that essentially all of `crayon` will be imbedded into the installed `testthat` this way. This can cause surprises. E.g. if there is a bug in `crayon`, and a new version is released to fix it, then you cannot simply fix the bug by installing a new version of `crayon`, since `testthat` does not import `crayon`. You need to rebuild `testthat`.

This of course all comes down to R's lack of build time dependencies. More precisely, this is not even a build time dependency, but it is an install time dependency. A build time dependency is what you need to build a source R package, e.g. roxygen2 to create the manual pages. `crayon` is needed when you install the source package.

Considering this, I think it is simpler to just import `green`, `yellow` and `red`. Or, we can still use `::`, but create `testthat_colours` at load time.